src/features/slackapi/slackapi-utils.ts

```diff
@@ -52,14 +55,24 @@ namespace SlackApiUtilities {
       billableHoursPercentage
     } = TimeUtilities.calculateWorkedTimeAndBillableHours(user);
 
-    return `     
-Hi ${name},
+    const customMessage = `
+Hi ${firstName},
 ${numberOfToday === 1 ? "Last friday" :"Yesterday"} (${displayDate}) you worked ${displayLogged} with an expected time of ${displayExpected}.
 ${message}
 Project time: ${displayProject}, Internal time: ${displayInternal}.
 Your percentage of billable hours was: ${billableHoursPercentage}% ${+billableHoursPercentage >= 75 ? ":+1:" : ":-1:"}
 Have a great rest of the day!
     `;
+    return {
+      message: customMessage,
+      name: name,
+      displayDate: displayDate,
+      displayLogged: displayLogged,
+      displayExpected: displayExpected,
+      displayProject: displayProject,
+      displayInternal: displayInternal,
+      billableHoursPercentage: billableHoursPercentage
+    };
```

Comment:
I'd create an own interface for this so you can make sure that the typing is correct everywhere where this data is used.